README.md

```diff
@@ -47,6 +47,88 @@ make notebook
 
 and selecting the pysyft kernel
 
+## Use the Docker image
+
+Instead of installing all the dependencies on your computer, you can run a notebook server (which comes with Pysyft installed) using [Docker](https://www.docker.com/). All you will have to do is start the container like this:
+
+```bash
+$ docker container run youben/pysyft:miniconda
```

Comment:
could you make this generic ie not just your user

Reply:
I can make it like this
```suggestion
$ docker container run pysyft/pysyft:miniconda
```
or this
```suggestion
$ docker container run openmined/pysyft:miniconda
```
but this require the creation of an organization in the Docker Hub, I can take care of the creation and the management of the organization, but if you prefer that someone from OpenMined do that, I will wait until he give me access to push the image. You chose.

Reply:
the openmined organization already exists on docker hub. I'm trying to figure out who has admin privileges to get this resolved

Reply:
yeah, I just saw that you already have an openmined/pysyft repo which haven't been update in a while. I could also work on that. Also @LaRiffle mentioned that we should use a more explanatory name (pysyft-notebook) which looks good (like pysonar and pysonar-notebook).

Reply:
@youben11 can you update this line?